src/generator.rs

```diff
@@ -157,13 +164,7 @@ fn write_contents(output_directory: &Path, filename: &str, contents: &str) -> Re
         .map(io::BufWriter::new)
         .and_then(|mut f| {
             f.write_all(b"# Automatically generated by ")?;
-            f.write_all(
-                env::current_exe()
-                    .as_deref()
-                    .unwrap_or_else(|_| Path::new(concat!(crate_name!(), " ", crate_version!())))
-                    .as_os_str()
-                    .as_bytes(),
-            )?;
+            f.write_all(*CURRENT_EXE)?;
```

Comment:
Same story here. The code is not longer, but it does bring in a new dep, and is harder to read. So we're making future maintenance harder, and we don't really gain much. In a debug build this adds 130kb of text.